### PR TITLE
Use docs.blockstack.org where appropriate

### PR DESCRIPTION
**Pull request by friedger**
This PR replaces documentation links with the corresponding links to docs.blockstack.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,14 +25,8 @@ To request some content for a Meetup you will be hosting, or to help clear up an
...
 - Resources for getting started...
-  * Any current documentation or pages that exist on the topic. Check _at least_ these spots:
-    * [Developer Getting Started](https://blockstack.org/devs)
-    * [Blockstack Forum](https://forum.blockstack.org/)
-    * [Tutorials](https://blockstack.org/tutorials)
-    * [Blockstack Core Tech Docs](https://github.com/blockstack/blockstack-core/tree/master/docs)
-    * [Blockstack Browser Tech Docs](https://github.com/blockstack/blockstack-browser/tree/master/docs)
-
-
+  * Any current documentation or pages that exist on the topic. Search _at least_ this spots:
```
Comment (moxiegirl) [inline README.md]: typo should be "this spot" or you could do "this documentation" . Thanks for this PR @friedger

**Review by moxiegirl — APPROVED** [review]
Thanks for these corrections @friedger

**Comment by moxiegirl** [comment]
@cuevasm Just to alert you once the small nit is picked...I think a merge is in order.  Thanks again @friedger

File: README.md (hunk 1 above)
Comment (friedger) [reply README.md]: Fixed

**Comment by moxiegirl** [comment]
Thanks @friedger !

**Pull request merged**